dev/doc/README.md

```diff
@@ -6,6 +6,40 @@ See the first section of [`INSTALL`](../../INSTALL).  Developers are
 recommended to use a recent OCaml version, which they can get through
 opam or Nix, in particular.
 
+## Configuring Dune caching and default verbosity
+
+There are several configuration settings that you can control globally
+with a Dune configuration file (see `man dune-config` to learn
+more). This file is generally located in `~/.config/dune/config` and
+should start with the version of the Dune language used, e.g.:
```

Comment:
```suggestion
There are several configuration settings that you can control globally
by creating a Dune configuration file (see `man dune-config` to learn
more). Dune uses the file `~/.config/dune/config` by default.  The file
should start with the version of the Dune language used, e.g.:
```

I didn't have such a file.  If users should always have such a file, please explain how it gets created.  If so, it would be nice if something would automatically create it.

Reply:
That is unfortunately system-specific. In normal operation the file is not needed.

Reply:
That == the path